app/services/flow/flows/provider_merits.rb

```diff
@@ -71,15 +71,22 @@ class ProviderMerits < FlowSteps
             if application.opponents.any?
               urls.providers_legal_aid_application_has_other_opponent_path(application)
             else
-              urls.new_providers_legal_aid_application_opponents_name_path(application)
+              Setting.opponent_organisations? ? urls.providers_legal_aid_application_opponents_types_path(application) : urls.new_providers_legal_aid_application_opponents_name_path(application)
             end
           end,
         },
+        opponents_types: {
+          path: ->(application) { urls.providers_legal_aid_application_opponents_types_path(application) },
+          forward: :opponents_names,
+          # TODO: currently this always forwards to the individual opponent page
```

Comment:
I don't think we need this todo. It will necessarily need to change as part of the ticket to add the organisation page I'd say

Reply:
👍 removed